update data formats according information from Solaris translators.
(it should be propagated to CLDR project probably)

diff -ruN cldr_core.orig//common/main/de.xml cldr_core/common/main/de.xml
--- cldr_core.orig//common/main/de.xml	2011-04-13 15:36:47.965991943 +0200
+++ cldr_core/common/main/de.xml	2011-04-13 15:41:58.101992007 +0200
@@ -1922,12 +1922,12 @@
 				<timeFormats>
 					<timeFormatLength type="full">
 						<timeFormat>
-							<pattern>HH:mm:ss zzzz</pattern>
+							<pattern>HH:mm:ss 'Uhr' zzzz</pattern>
 						</timeFormat>
 					</timeFormatLength>
 					<timeFormatLength type="long">
 						<timeFormat>
-							<pattern>HH:mm:ss z</pattern>
+							<pattern>HH:mm:ss 'Uhr' z</pattern>
 						</timeFormat>
 					</timeFormatLength>
 					<timeFormatLength type="medium">
